src/frontend/src/components/base/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { tv } from "tailwind-variants";
+
+export const checkboxStyles = tv({
+    slots: {
+        root: "flex items-start gap-3",
+        box: "flex h-6 w-6 flex-shrink-0 items-center justify-center rounded-[var(--spacing-3)] border-2 transition",
+        labelContainer: "flex flex-col gap-1",
+        label: "text-rg font-bold text-gray-900",
+        description: "text-sm text-gray-600",
+    },
```

Comment:
Nice! The checkbox example is where slots is finally clicking for me! One thing I'm thinking about is we now have two styling patterns: conditionally styled variants and static variants. If a style is static what do you think about keeping that in `base` and then only conditionally styled elements are defined in `slots`? That might keep our different base component style files more cohesive and be easier to get oriented for others jumping in

Reply:
Documenting that Aaron and I discussed in a call which helped clarify my understanding of slots! Slots isn't necessarily _just_ to support conditionally styling a specific element, that functionality is an additional benefit to slots which fundamentally organize a component's styling using keys to target specific elements to style. It doesn't benefit us to separate out element styles that are static vs dynamic, instead if a component needs targeted styling (ie is more than one element) we should use the above slots format to style. If we don't need to style different parts of a component we should continue with `base` (like `TextField` styles).